src/Components/__tests__/Utils/addressFormTLHelpers.ts

```diff
@@ -0,0 +1,85 @@
+import { screen, fireEvent } from "@testing-library/react"
+import { Address } from "Components/AddressForm"
+export const validAddress: Address = {
+  name: "Erik David",
+  addressLine1: "401 Broadway",
+  addressLine2: "",
+  city: "New York",
+  region: "NY",
+  postalCode: "15601",
+  phoneNumber: "5555937743",
+  country: "US",
+}
+
+export const fireChangeEvent = (input: HTMLElement, value: string) => {
+  fireEvent.change(input, {
+    target: { value: value },
+  })
+}
+
+export const fillAddressFields = (address: Address = validAddress) => {
+  fireChangeEvent(screen.getByPlaceholderText("Full name"), address.name)
+  fireChangeEvent(screen.getByLabelText("Country"), address.country || "US")
+
+  fireChangeEvent(
+    screen.getByPlaceholderText("Street address"),
+    address.addressLine1 || ""
+  )
+  fireChangeEvent(
+    screen.getByPlaceholderText("Apt, floor, suite, etc."),
+    address.addressLine2 || ""
+  )
+
+  fireChangeEvent(screen.getByPlaceholderText("City"), "New York")
+  fireChangeEvent(
+    screen.getByPlaceholderText("ZIP/postal code"),
+    address.postalCode || ""
+  )
+  fireChangeEvent(
+    screen.getByPlaceholderText("State, province, or region"),
+    address.region || ""
+  )
+
+  fireChangeEvent(
+    screen.getAllByPlaceholderText(
+      "Add phone number including country code"
+    )[0],
+    address.phoneNumber || ""
+  )
+}
```

Comment:
We should co-locate this file with the address form file/test. Here it is buried and prematurely abstracted, when it only has to do with one thing (the address form)

Reply:
I was trying to mirror the structure of the current tests and helpers, but maybe a bigger break or new file is a good idea.

Reply:
If you stick with colocation in the place that the helper is being called from (until said helper is used > 3 times), and avoid abstraction as much as humanly possible, you'll always win 😄